frappe/public/js/frappe/form/grid.js

```diff
@@ -514,6 +514,7 @@ frappe.ui.form.GridRow = Class.extend({
 		this.wrapper.addClass("grid-row-open");
 		frappe.ui.scroll(this.wrapper, true, 15);
 		me.frm.script_manager.trigger(me.doc.parentfield + "_on_form_rendered");
+		me.frm.script_manager.trigger("form_rendered", me.doc.doctype, me.doc.name);
```

Comment:
Should be `form_render`. The legacy `..ed` shouldn't be carried forward! Else, we would've had functions like `refreshed` and `on_submitted`, etc.